docs/articles/documentation/reference/test-api/test/timeouts.md

```diff
@@ -0,0 +1,37 @@
+---
+layout: docs
+title: Test.timeouts Method
+permalink: /documentation/reference/test-api/test/timeouts.html
+---
+# Test.timeouts Method
+
+Sets timeouts for the test.
+
+```text
+test.timeouts({
+    [pageLoadTimeout]
+    [, pageRequestTimeout]
+    [, ajaxRequestTimeout]
+}) → this
+```
+
+Parameter                         | Type   | Description
+--------------------------------- | ------ | ---------------------------------------------------------------------------
+`pageLoadTimeout` *(optional)*    | number | The time (in milliseconds) passed after the `DOMContentLoaded` event, within which TestCafe waits for the `window.load` event to fire. See [pageLoadTimeout](../../configuration-file.md#pageloadtimeout).
+`pageRequestTimeout` *(optional)* | number | The time (in milliseconds) to wait for HTML pages. See [pageRequestTimeout](../../configuration-file.md#ajaxrequesttimeout).
```

Comment:
You can't *wait* for an HTML page. You can wait for a response to your request.
```suggestion
`pageRequestTimeout` *(optional)* | number | Maximum web server response time (in milliseconds). See [pageRequestTimeout](../../configuration-file.md#ajaxrequesttimeout).
```

Reply:
> You can't wait for an HTML page. You can wait for a response to your request.

`Wait for an HTML (to be served)` == `wait for a response to a request for an HTML page`

> Maximum web server response time

Doesn't indicate that only requests for webpages (when the browser requests HTML from the server) are affected. A fetch request (for example) is still made to a web server, yet has nothing to do with this option. 

```suggestion
`pageRequestTimeout` *(optional)* | number | Maximum time (in milliseconds) for the web server to serve an HTML page. See [pageRequestTimeout](../../configuration-file.md#ajaxrequesttimeout).
```

Gets a bit repetitive. So I'd say `to wait for HTML pages` is good enough.

Reply:
Your new suggestion is better and less colloquial, in a good way tbh. Thanks!